website/content/docs/internals/recommended-patterns.mdx

```diff
@@ -246,7 +246,7 @@ A baseline provides insight into current utilization and thresholds. Telemetry m
 Telemetry information can also be streamed directly from Vault to a range of metrics aggregation solutions and 
 saved for aggregation and inspection. 
 
-- [Vault usage metrics](/vault/tutorials/monitoring/usage-metrics)
+- [Vault client usage](/vault/tutorials/monitoring/usage-metrics)
```

Comment:
```suggestion
- [Vault client usage](/vault/tutorials/monitoring/client-usage)
```

Reply:
Amending based on our conversation in slack, this should be the following because the `vault/tutorials` is part of the redirect mapping which is outdated
```suggestion
- [Vault client usage](/vault/docs/concepts/client-count/client-usage)
```